byoqm/runner/runner.py

```diff
@@ -92,25 +93,42 @@ def _run_metrics(self) -> Dict:
             sys.modules[metric] = module
             spec.loader.exec_module(module)
             module.metric.coordinator = self._coordinator
-            results[metric] = int(module.metric.run())
+            results[metric] = module.metric.run()
         logging.info("Finished running metrics")
         return results
 
+    def _generate_violations_table(self, results: Dict, time: str):
+        list_of_violations = []
+        for _, violations in results.items():
+            if type(violations) is list:
+                list_of_violations = list_of_violations + violations
+        violations = pd.DataFrame(
+            list_of_violations, columns=["type", "file", "start", "end"]
+        )
+        violations.attrs = {"model": self._model_name, "root": self._src_root}
+        file_location = Path(self._output_dir / Path(time + "_location.csv"))
+        violations.to_csv(file_location)
```

Comment:
Should this be in the output directory, or a separate directory? I can imagine it getting confusing if we have both types of files in the output directory (even though their names are different)

Reply:
I think that is something to discuss in the group